plugins/azure/redisCache/redisCacheVNetIntegrated.js

```diff
@@ -0,0 +1,53 @@
+var async = require('async');
+var helpers = require('../../../helpers/azure');
+
+module.exports = {
+    title: 'Redis Cache VNet Integrated',
+    category: 'Redis Cache',
+    domain: 'Databases',
+    description: 'Ensures that premium Redis Cache has VNet integrated.',
+    more_info: 'Ensuring VNet deployment for Redis Cache provides enhanced security and isolation. When VNet is combined with restricted NSG policies, it helps reducing the risk of data exfiltration.',
+    recommended_action: 'Ensure VNet (private access) is integrated for premium Redis Cache.',
+    link: 'https://learn.microsoft.com/en-us/azure/azure-cache-for-redis/cache-network-isolation',
+    apis: ['redisCaches:listBySubscription'],
```

Comment:
```suggestion
    apis: ['redisCaches:listBySubscription'],
```
add realtime triggers